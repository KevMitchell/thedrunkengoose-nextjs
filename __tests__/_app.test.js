import { render } from '@testing-library/react'
import App from '../pages/_app'

const component = () => <div />

test('renders navigation', () => {
  const { getAllByRole } = render(<App Component={component} />)
  const navigationLinks = getAllByRole(
    /link/
  )
  expect(navigationLinks).toMatchSnapshot()
})

test('renders the footer', () => {
  const { getByText } = render(<App Component={component} />)
  const textElement = getByText(
    /Copyright Â© Kevin Mitchell, 2020/
  )
  expect(textElement).toBeInTheDocument()
})
