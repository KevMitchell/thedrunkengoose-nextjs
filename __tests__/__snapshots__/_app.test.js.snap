// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navigation 1`] = `
Array [
  <a
    class="jsx-1730214407"
    href="/about"
  >
    About us
  </a>,
  <a
    class="jsx-1730214407"
    href="/menu"
  >
    Menu
  </a>,
]
`;
